velox/dwio/common/Options.h

```diff
@@ -166,6 +166,7 @@ class RowReaderOptions {
   std::shared_ptr<UnitLoaderFactory> unitLoaderFactory_;
 
   TimestampPrecision timestampPrecision_ = TimestampPrecision::kMilliseconds;
+  const date::time_zone* sessionTimezone_{nullptr};
```

Comment:
Can we find another way (e.g. `FormatData` or `StripeStreams`) to pass it down to column readers?  Should not have redundancy in interface object like this.

Reply:
Now `sessionTimezone` has been refactored to be taken from `StripeStreams`'s `readState_->readerBase->getReaderOptions().getSessionTimezone()`.